mscore/events.cpp

```diff
@@ -879,6 +889,7 @@ void ScoreView::changeState(ViewState s)
                   break;
             case ViewState::ENTRY_PLAY:
```

Comment:
There needs to be a `break;` here.

Reply:
Thanks for the review. Since I'm relatively new to C++, could you explain why this is?

Reply:
C++ uses the concept of "fall through" to handle multiple cases in a switch statement. If you do not want the handling of one case to fall through to the next one, you must use a `break;` to exit the switch statement. Since originally there was no code to be executed for any of these cases, there was no reason to insert a `break;` between each of them.

Reply:
Great, that makes sense, thanks.